src/View.php

```diff
@@ -215,11 +215,9 @@ public function __construct($label = null, $class = null)
      *
      * @return Model
      */
-    public function setModel(Model $m)
+    public function setModel(Model $model)
     {
-        $this->model = $m;
-
-        return $m;
+        return $this->model = $model;
```

Comment:
I think 2 lines are much better to read - you assign and then you return.

Reply:
reverted